File: p1/Solution.py
# returns true if the given int is a multiple of either 3 or 5 and false otherwise
def isMultiple(n):
    return n % 3 == 0 or n % 5 == 0

if __name__ == "__main__":
    sum = 0                             
    for i in range(1000):               
        if isMultiple(i) : sum += i     # updates the sum if i is a multiple
    print("Ans: " + str(sum))           

File: p1/Solution.java
public class Solution{
    public static void main(String[] args){
        int sum = 0;                            
        for(int i = 1; i < 1000; i++){         
            if(isMultiple(i)){sum += i;}        // updates the sum if i is a multiple
        }
        System.out.println("Ans: " + sum);
    }

    /* returns true if the given int is either a multiple of 3 or 5 and false otherwise */
    private static boolean isMultiple(int n){
        return n % 3 == 0 || n % 5 == 0;
    }
}